client/my-sites/exporter/export-media-card/index.js

```diff
@@ -56,6 +57,12 @@ class ExportMediaCard extends Component {
 											'Download all the media library files (images, videos, audio and documents) from your site.'
 										) }
 									</h2>
+									<h2 className="export-media-card__warning">
+										{ translate(
+											'Depending on your media library size and/or connection speed you might need to use a download manager.'
+										) }
+										<InlineSupportLink supportContext="export-media-library" showIcon={ false } />
```

Comment:
We should include a full stop (period) after "Learn more" like we do in the subheading:

<img width="757" alt="Screen Shot 2021-11-18 at 1 23 55 PM" src="https://user-images.githubusercontent.com/2124984/142474507-9c0818dd-517a-41c5-a2d1-ae2ee307757b.png">

A good code example of this in action is here: 

https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/my-sites/types/main.jsx#L33-L40

This will also fix the missing space problem I noticed:
<img width="734" alt="Screen Shot 2021-11-18 at 1 22 24 PM" src="https://user-images.githubusercontent.com/2124984/142474815-71cf6e87-9757-407a-9c38-083387e96c3c.png">

Reply:
Thanks for the interpolation example!